project/Dependencies.scala

```diff
@@ -59,20 +59,11 @@ object Dependencies {
   }
 
   val scalaTestVersion = Def.setting {
-    if (scalaVersion.value.startsWith("3.")) {
-      "3.2.9"
-    } else {
-      "3.1.4"
```

Comment:
Earlier (https://github.com/akka/akka/issues/29368#issuecomment-679843504) we decided not to update scalatest yet.